UserInterface/SwitchDemos/SwitchDemos/SwitchDemos/SwitchCodePage.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Xamarin.Forms;
+
+namespace SwitchDemos
+{
+    public class SwitchCodePage : ContentPage
+    {
+        Label switchStateLabel;
+
+        public SwitchCodePage()
+        {
+            Title = "Switch Code Demo";
+
+            switchStateLabel = new Label
+            {
+                Text = "Switch state is OFF",
+                HorizontalOptions = LayoutOptions.CenterAndExpand,
+                VerticalOptions = LayoutOptions.CenterAndExpand
+            };
+
+            Switch switchControl = new Switch
+            {
+                IsToggled = false,
+                OnColor = Color.Green,
```

Comment:
Maybe go with a different colour, as it's pretty similar to the colour used on iOS already.